mkdocs.yml

```diff
@@ -100,6 +100,9 @@ nav:
       - page/gallery.md
   - Changelog:
       - page/CHANGELOG.md
+  - Tutorials:
+      - C/C++:
+          - 'Configuring Emacs a C/C++ IDE': tutorials/CPP-guide.md
```

Comment:
Will we need this kind of level of segregation? what about just:
```suggestion
  - Tutorials:
      - 'Configuring Emacs a C/C++ IDE': tutorials/CPP-guide.md
```

Reply:
I merged your proposal - we may revisit it later.